packages/schematics/angular/resolver/schema.json

```diff
@@ -25,6 +25,11 @@
       "description": "When true (the default), creates the new files at the top level of the current project.",
       "default": true
     },
+    "functional": {
+      "type": "boolean",
+      "description": "When true, creates the resolver as a `ResolveFn`.",
+      "default": true
```

Comment:
This also looks like a breaking change as there is a change in behaviour.

Reply:
The changes to the defaults were done in separate commits. If those need to be considered breaking, I'm happy to drop those commits. That said, it will be a bit weird for a bit in the CLI that it will generate deprecated code by default.